src/message/buttons/urlButton.js

```diff
@@ -35,7 +35,13 @@ export default class UrlButton extends Button {
         return this;
     }
 
+    setMessengerExtensions(value = false) {
```

Comment:
We should probably then change it to useMessengerExtensions, if thats true, and disable if we do not need it